docs/01_general/05_why.md

```diff
@@ -3,6 +3,31 @@ title: Why
 path: /docs/why
 ---
 
-# Why
+# Why was Strawberry created?
 
-Documentation coming soon
+Strawberry has been inspired by dataclasses[^1] and one of its goals is to
+provide a great developer experience for both GraphQL beginners and advanced
+users.
+
+In addition to that we really want to create and foster a nice and welcoming
+community with people using GraphQL in python.
+
+# Why should you use Strawberry?
+
+> Note: at the moment Strawberry is still in early development, so there's a
+> change of things suddenly changing, but hopefully the public API is stable
+> enough.
+
+Thanks to type hints and the decorator syntax inspired by dataclasses,
+Strawberry provides a nice developer experience that will help writing better
+GraphQL APIs while also helping finding bugs when using type checkers like MyPy.
+
+We also are going to provide useful features and integrations, for example we
+provide support for Apollo Federation, File Uploads, Permissions and integration
+with popular frameworks like Django, ASGI and Flask.
+
+Finally we try to fix bugs and provide help via GitHub or our discord channel,
```

Comment:
```suggestion
Finally, we try to fix bugs and provide help via GitHub or our Discord server,
```

We might want to have a link the discord server here, as well?

Reply:
good idea, I'll see if I can make a `discord.strawberry.rocks` link tomorrow :)